internal/services/automation/automation_webhook_resource.go

```diff
@@ -98,7 +99,7 @@ func resourceAutomationWebhook() *pluginsdk.Resource {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
 				ForceNew:     true,
-				Computed:     true,
+				Computed:     !features.FourPointOhBeta(),
```

Comment:
Can we set an appropriate default here as well?

Reply:
this is generated so we can't use a default here